iree/compiler/Conversion/LinalgToSPIRV/LinalgTileAndFusePass.cpp

```diff
@@ -506,6 +527,47 @@ static void applyVectorTransformation(FuncOp funcOp) {
   }
 }
 
+//====---------------------------------------------------------------------===//
+// Patterns to tile convolution window dimensions
+//====---------------------------------------------------------------------===//
+
+/// Populates `patterns` with patterns that tiles a linalg.conv along filter's
+/// height, width, input channel dimensions with tile sizes 1, 1, 4.
+static void populateTilingConvFilterPatterns(MLIRContext *context,
+                                             OwningRewritePatternList &patterns,
+                                             linalg::LinalgMarker marker) {
+  auto getTileSizeFn = [](OpBuilder &builder, Operation *op) {
+    Location loc = op->getLoc();
+    auto zero = builder.create<ConstantIndexOp>(loc, 0);
+    auto one = builder.create<ConstantIndexOp>(loc, 1);
+    auto four = builder.create<ConstantIndexOp>(loc, 4);
+    // linalg.conv has 7 iterators.
+    SmallVector<Value, 4> tileSizes(7, zero);
+    // The last 3 are for the filter.
+    tileSizes[4] = four;                // input channel
+    tileSizes[5] = tileSizes[6] = one;  // filter height/width
+    return tileSizes;
+  };
+
+  SmallVector<unsigned, 8> loopOrder = {
```

Comment:
Maybe even add `loopOrder` to `LaunchConfig`?

Reply:
SGTM. But I think it will require a larger scope change. I'll wait for your https://github.com/google/iree/pull/4008 to land first and then address this.